encoder/encoder.go

```diff
@@ -0,0 +1,19 @@
+package encoder
+
+import (
+	"github.com/smartcontractkit/ocr2keepers/pkg/types"
+)
+
+type Config struct{}
+
+type Encoder interface {
```

Comment:
One pattern that I want to avoid is defining large interfaces in general and defining interfaces in the package in which they are implemented. I prefer defining interfaces in the packages in which they are consumed and the smaller the better.

If this is temporary just for tracking progress, disregard my comments. :)

Reply:
Yeah totally, I'm happy to move these interfaces into their consuming packages when we get to integrating these new components, but I actually like having to import from the interface package since I find it forces you to consider the relationship between packages more? But totally, these will be moved out to wherever makes sense later on 👍